### DISCUSSION TITLE
Option, "heroes: allow buy a spellbook from Shrines", set same cost for all Shrines

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3384] [Ideas]
I suggest to set the same cost of 500 Gold for the different Shrines as in a Mage Guild (instead of a different price in each type of Shrine).

In fheroes2:

![Buy Book in shrine 001](https://user-images.githubusercontent.com/43583994/117667852-a3c13300-b1a5-11eb-94c9-e152142a8e1f.png)

![Buy Book in shrine 002](https://user-images.githubusercontent.com/43583994/117667867-a6bc2380-b1a5-11eb-91ea-c36baaa5d852.png)

**Edit 2021/05/16.**

Related issue: https://github.com/ihhub/fheroes2/issues/1220.

**Comment by Kromster80** [comment]


**Reply by LeHerosInconnu** [reply]
@Kromster80.

> Different price has a nice touch to it - player is slightly penalized for not thinking about buying the book in advance :-)

Why penalize the player?
The game should bring fun, not frustration.
And buying a Magic Book twice or three times its price is a real ripoff.

Not to mention that a Barbarian will take the Magic Book by force without paying for it...
...and will expose the corpses of the victims as an example and a threat to anyone who wants to do this. :)

@Branikolog.

I saw your dark side. :D

**Reply by Branikolog** [reply]
@LeHerosInconnu 
I'm sorry. I cannot handle my hunger for fresh green goblin meat. :P

**Comment by LeHerosInconnu** [comment]
This option has been removed from fheroes2.
This option is planned in my mod: https://github.com/ihhub/fheroes2/discussions/7832.
I close the discussion.
